float64 scan_angle_range
---
bool started_scan